---
title: "[AC-4690] Android : Error when closing a window in a ScrollableView."
---
<table>
<tr><th>Type</th><td>Bug</td></tr>
<tr><th>Priority</th><td>n/a</td></tr>
<tr><th>Status</th><td>Resolved</td></tr>
<tr><th>Resolution</th><td>Invalid</td></tr>
<tr><th>Resolution Date</th><td>2016-12-22T15:09:13.000+0000</td></tr>
<tr><th>Affected Version/s</th><td>n/a</td></tr>
<tr><th>Fix Version/s</th><td>n/a</td></tr>
<tr><th>Components</th><td>Titanium SDK & CLI</td></tr>
<tr><th>Labels</th><td>n/a</td></tr>
<tr><th>Reporter</th><td>Dongwoo Gim</td></tr>
<tr><th>Assignee</th><td>Shak Hossain</td></tr>
<tr><th>Created</th><td>2016-12-22T08:06:38.000+0000</td></tr>
<tr><th>Updated</th><td>2016-12-22T15:09:13.000+0000</td></tr>
</table>

<h3>Description</h3>

An error when closing a window in a ScrollableView.
Using Ti SDK 6.1.0.v20161221030858

But 6.1.0.v20161205110006 is ok.
It occurs in later versions of this.

<code><pre>
[ERROR] TiApplication: (main) [232,481087] Sending event: exception on thread: main msg:java.lang.NullPointerException: Attempt to invoke virtual method 'boolean org.appcelerator.kroll.KrollProxy.hasProperty(java.lang.String)' on a null object reference; Titanium 6.1.0,2016/12/21 03:09,undefined
[ERROR] TiApplication: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean org.appcelerator.kroll.KrollProxy.hasProperty(java.lang.String)' on a null object reference
[ERROR] TiApplication: 	at org.appcelerator.titanium.util.TiUIHelper.requestSoftInputChange(TiUIHelper.java:1028)
[ERROR] TiApplication: 	at org.appcelerator.titanium.view.TiUIView$4.run(TiUIView.java:1174)
[ERROR] TiApplication: 	at android.os.Handler.handleCallback(Handler.java:739)
[ERROR] TiApplication: 	at android.os.Handler.dispatchMessage(Handler.java:95)
[ERROR] TiApplication: 	at android.os.Looper.loop(Looper.java:148)
[ERROR] TiApplication: 	at android.app.ActivityThread.main(ActivityThread.java:5417)
[ERROR] TiApplication: 	at java.lang.reflect.Method.invoke(Native Method)
[ERROR] TiApplication: 	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
[ERROR] TiApplication: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
[ERROR] D/        ( 1556): HostConnection::get() New Host Connection established 0x7f4439496cc0, tid 3177

</pre></code>


<h3>Comments</h3>

<ol>
<li>Dongwoo Gim 2016-12-22

   Sorry. this error is solved in 6.1.0.v20161222003214</li>
<li>Hans Kn√∂chel 2016-12-22

   Thx!</li>
</ol>


<p><a href="/AC/AC-4690.json">JSON Source</a></p>